book/src/build_and_install.md

```diff
@@ -0,0 +1,25 @@
+# Building and Installing
+
+## Prerequisites
```

Comment:
LLVM 11 is also needed
On Linux, the package manager version is fine
On windows you need a special build : https://github.com/ghaith/llvm-package-windows/releases/tag/v11.0.1

Reply:
You also need build tools (build-essential on ubuntu) and liblz-dev

Reply:
Added!